core/domain/summary_services_test.py

```diff
@@ -764,3 +766,50 @@ def test_for_recently_published_explorations(self):
             recently_published_exploration_summaries[1]['title'], 'New title')
         self.assertDictContainsSubset(
             test_summary_3, recently_published_exploration_summaries[0])
+
+
+class ActivityReferenceValidationTests(test_utils.GenericTestBase):
```

Comment:
I don't think I agree with this suite name. Firstly, I'd expect validation tests for activity references to go in its corresponding domain test class. Secondly, this isn't really testing validation. It seems to currently be testing ``summary_services.require_activities_to_be_public``. Perhaps ``ActivityReferenceAccessCheckerTests``